### DISCUSSION TITLE
How can I configure standalone Milvus to use external storage (not system disks) for the embedded etcd and minio files?

### DISCUSSION THREAD

**Discussion by liorf95** [milvus-io/milvus#35453] [Q&A and General discussion]
What is the needed configuration for that in the milvus.yaml?

**Comment by yhmo** [comment]
If the milvus is deployed with embed etcd: https://milvus.io/docs/install_standalone-docker.md, all the data files and etcd data are stored in the path ./volumes/milvus. I think you can mount this path to external storage.
Wait @LoveEachDay to confirm.

**Comment by xiaofan-luan** [comment]
> What is the needed configuration for that in the milvus.yaml?

all the config is in milvus.yaml

you need to change 
rocksmq.path
etcd.data and localStorage.path to the right directory

**Reply by senthil-bungee** [reply]
Thanks, Does the path should represent prefix key in the AWS? Some sample config with values & assume role will be good.

**Reply by xiaofan-luan** [reply]
If you are running with docker compose mode, with S3 or minio

you need to config minio.bucketName and minio.rootPath
